<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>968</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MSFS Dashboard</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,2">
        <item>
         <widget class="QGroupBox" name="connectionGroup">
          <property name="title">
           <string>Connection</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="connectButton">
             <property name="text">
              <string>Connect</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="statusLight">
             <property name="minimumSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border-radius: 10px; background-color: red;</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="controlsGroup">
          <property name="title">
           <string>Controls</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,1">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QPushButton" name="gearButton">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Toggle Gear</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="dataGroup">
               <property name="title">
                <string>Data</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <property name="spacing">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="gearLabel">
                  <property name="text">
                   <string>Gear: ---%</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="gearCenterLabel">
                  <property name="text">
                   <string>C: ---%</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="gearLeftLabel">
                  <property name="text">
                   <string>L: ---%</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="gearRightLabel">
                  <property name="text">
                   <string>R: ---%</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="gearWarningLabel">
                  <property name="styleSheet">
                   <string notr="true">color: red; font-weight: bold;</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="brakeButton">
             <property name="text">
              <string>Parking Brake</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="apButton">
             <property name="text">
              <string>Autopilot</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4" stretch="3,1">
        <item>
         <layout class="QHBoxLayout" name="horizonLayout" stretch="1,1">
          <item>
           <widget class="AttitudeIndicator" name="attitudeIndicator" native="true"/>
          </item>
          <item>
           <widget class="Compass" name="compass" native="true"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="engineGroup">
          <property name="title">
           <string>Engines</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="1" column="0">
            <widget class="QPushButton" name="eng1Button">
             <property name="text">
              <string>Start Eng 1</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="RpmIndicator" name="rpmIndicator1" native="true"/>
           </item>
           <item row="0" column="1">
            <widget class="RpmIndicator" name="rpmIndicator2" native="true"/>
           </item>
           <item row="0" column="2">
            <widget class="RpmIndicator" name="rpmIndicator3" native="true"/>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="eng2Button">
             <property name="text">
              <string>Start Eng 2</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="eng3Button">
             <property name="text">
              <string>Start Eng 3</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="RpmIndicator" name="rpmIndicator4" native="true"/>
           </item>
           <item row="1" column="3">
            <widget class="QPushButton" name="eng4Button">
             <property name="text">
              <string>Start Eng 4</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>968</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMSFS20_24_DashBoard">
    <property name="title">
     <string>MSFS20/24 DashBoard</string>
    </property>
    <addaction name="actionsource_code"/>
   </widget>
   <addaction name="menuMSFS20_24_DashBoard"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionsource_code">
   <property name="text">
    <string>source code</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AttitudeIndicator</class>
   <extends>QWidget</extends>
   <header>AttitudeIndicator.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Compass</class>
   <extends>QWidget</extends>
   <header>Compass.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RpmIndicator</class>
   <extends>QWidget</extends>
   <header>RpmIndicator.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
